source/server/options_impl.cc

```diff
@@ -11,6 +11,10 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
+#if defined(__APPLE__)
+#define uint64_t unsigned long
```

Comment:
Finally figured this out. TCLAP provides implementations of `ArgTraits<T>` for various types, but the ones for `unsigned long long` and `long long` are gated by `#ifdef HAVE_LONG_LONG`, which isn't set. I think they were expecting users of libraries to use autoconf which would detect this and make the necessary #define. I'll post it with a few other things tomorrow (split out in a new PR).